openvino_tensorflow/ie_basic_engine.cc

```diff
@@ -30,11 +36,23 @@ void IE_Basic_Engine::infer(
     std::vector<std::string>& output_names,
     std::vector<std::shared_ptr<IETensor>>& hoisted_params,
     std::vector<std::string>& param_names) {
+  int64_t start_ns = GetCurrentTimeNanos();
```

Comment:
How about doing this like below? So, if we are not printing the timing, we don't need to call GetCurrentTimeNanos().

int64_t start_ns;
if (<inference_timing_enabled>) {
    start_ns = GetCurrentTimeNanos();
}